Sources/PaymentButtons/Assets.xcassets/Redesign Logos/paypal_credit.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
In the [VoiceOver PR](https://github.com/paypal/paypal-ios/pull/236), I took the approach of using the `.imageset` name to determine which logo is being displayed. If it was prefixed with `paypal_`, I assumed it only had the PP logo / PP text, while if it was prefixed with `credit_` then it contained the PP logo & credit text.

If we want to use that pattern, then we would need to update the names of these assets to move `credit_` to the front.

I am down though for a better/more clear way for us to know in code what content each image type has. Is there a more clear naming convention we can use? Also the logic which appends the `imageAssetString` together in `ImageAsset.swift` is a little confusing IMO. It might be more clear to just have the fully hard coded strings that match the `.imageset` filenames vs concatenating. What do folks think?

Reply:
I also had trouble with the logic when I was updating the file and was going off of what was originally there. I assume the concatenation was to reduce the amount of duplicate code. I'd love for them to be hardcoded to match `.imageset` filenames rather than concatenating so I'll update to that!

Reply:
![Screenshot 2024-01-04 at 3 04 38 PM](https://github.com/paypal/paypal-ios/assets/127455800/46092b7a-015f-4a52-a3b4-fd35bf62a3f4)

Here are the changes I made

Reply:
Awesome! Yeh, I agree - a bit easier to read and less error prone.

🏈 After we get the main parts of this work done for QL, would be great to add some UI/snapshot tests for these buttons